release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3332,6 +3332,41 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.16.9
+[id="ocp-4-16-9_{context}"]
+=== RHBA-2024:5757 - {product-title} {product-version}.9 bug fix update
+
+Issued: 28 August 2024
+
+{product-title} release {product-version}.9 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:5757[RHBA-2024:5757] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:5760[RHBA-2024:5760] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.9 --pullspecs
+----
+
+[id="ocp-4-16-9-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the installation program failed to install an {product-title} cluster in the `eu-es` (Madrid, Spain) region on a {ibm-power-server-title} platform that was configured as an `e980` system type. With this release, the installation program no longer fails to install a cluster in this environment. (link:https://issues.redhat.com/browse/OCPBUGS-38502[*OCPBUGS-38502*])
+
+// https://issues.redhat.com/browse/OCPBUGS-36227 is private and has a dup that is also private. https://issues.redhat.com/browse/OCPBUGS-36188
```

Comment:
No reply from SME so I'll skip these dup bugs as they are both marked as private.

Reply:
Not sure where this is documented internally, but we are not supposed to document RH Employee only bugs.

Reply:
Yeah, the duplicate bugs might be an indication that one of the bugs would be cleaned of customer info for the purposes of making the bug public. I wanted to check with the SME but I've received no reply.